manifests/server.pp

```diff
@@ -1,15 +1,45 @@
-#
 # @summary installs the hyperglass looking glass
 #
 # @param manage_depended_services if true, installs all other services that hyperglass requires, like redis, yarn, nginx, python
+# @param devices hash containing all the devices hyperglass can connect to. Defaults to demo data so the service starts properly.
```

Comment:
I prefer putting the description on a separate line.  IMO it's more readable.  Style-guide says do either though (and then has all examples using separate line)
https://puppet.com/docs/puppet/6.17/puppet_strings_style.html#parameters
> A description of what the parameter does. This may be either on the same line as the @param tag or on the next line, indented with two spaces.